packages/cli/src/util/client.ts

```diff
@@ -25,7 +29,7 @@ export async function createMedplumClient(options: MedplumClientOptions): Promis
     medplumClient.setAccessToken(accessToken);
   }
 
-  if (clientId && clientSecret) {
+  if (clientId && clientSecret && !(profileObject?.authType === 'jwt-bearer')) {
```

Comment:
This block was always a kludge.  It's incorrect.

I think the original intent was for it to be `profile.authType === 'client_credentials'`

Reply:
I think it's still needed for basic so I changed the conditional to include an `else if profile.authType` === 'basic' 

I also changed client.test.ts `'setBasicAuth and startClientLogin'` to add a profile with an auth type of client_credentials